### PR TITLE
bug 1544032 clarifying embedded etcd

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1544032

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@gpei, will you PTAL?

**Comment by kalexand-rh** [comment]
@gpei, I fixed the directory. Will you please take another look?

**Comment by kalexand-rh** [comment]
Gayoun Pei approved this change on the source bug.

@openshift/team-documentation PTAL

File: install_config/downgrade.adoc [hunk 1]
```diff
@@ -114,7 +115,7 @@ remove the following packages:
     atomic-openshift-docker-excluder
 ----
 
-. If you are using external etcd, also remove the *etcd* package:
+. If you use a separate etcd cluster, also remove the *etcd* package:
```
Comment (mburke5678) [inline install_config/downgrade.adoc]: @kalexand-rh  
Maybe someone who knows etcd will know this, but are _separate etcd cluster that runs on different nodes,_ and _separate etcd cluster,_ the same?

**Comment by mburke5678** [comment]
@kalexand-rh  One question. Otherwise LGTM

File: install_config/downgrade.adoc (hunk 1 above)
Comment (kalexand-rh) [reply install_config/downgrade.adoc]: Ah, I see what you mean. I'm going to change one of them. Thanks!

**Pull request merged**